### PR TITLE
Rejigger project structure; add necessary publishing files.

### PR DESCRIPTION
**Pull request by twaddington**
This isn't quite working yet, but it's close.

I'm having trouble following what your update script is doing. Running `setup.py develop` or `install` works fine. You can then call `android-sdk-updater` or `android-sdk-updater --help` from the command-line and get output as expected.

However, I'm seeing results like this:

```
(android-sdk-updater)pierrot:src tristan$ android-sdk-updater 
Scanning /Users/tristan/.android-sdk for installed packages...
Package parse failed: /add-ons/addon-google_apis-google-15
Package parse failed: /add-ons/addon-google_apis-google-16
Package parse failed: /add-ons/addon-google_apis-google-17
Package parse failed: /add-ons/addon-google_apis-google-18
Package parse failed: /add-ons/addon-google_apis-google-19
Package parse failed: /add-ons/addon-google_apis-google-21
Package parse failed: /add-ons/addon-google_apis-google-22
Package parse failed: /add-ons/addon-google_apis-google-23
Package parse failed: /add-ons/addon-google_apis_x86-google-19
Package parse failed: /add-ons/addon-google_gdk-google-19
Package parse failed: /build-tools/17.0.0
Package parse failed: /build-tools/18.0.1
Package parse failed: /build-tools/18.1.0
Package parse failed: /build-tools/18.1.1
Package parse failed: /build-tools/19.0.0
Package parse failed: /build-tools/19.0.1
Package parse failed: /build-tools/19.0.2
Package parse failed: /build-tools/19.0.3
Package parse failed: /build-tools/19.1.0
Package parse failed: /build-tools/21.0.0
Package parse failed: /build-tools/21.0.1
Package parse failed: /build-tools/21.0.2
Package parse failed: /build-tools/21.1.0
Package parse failed: /build-tools/21.1.1
Package parse failed: /build-tools/21.1.2
Package parse failed: /build-tools/22.0.0
Package parse failed: /build-tools/22.0.1
Package parse failed: /build-tools/23.0.0
Package parse failed: /build-tools/23.0.1
Package parse failed: /build-tools/23.0.2
Package parse failed: /build-tools/android-4.4W
Package parse failed: /docs
Package parse failed: /extras/android/m2repository
Package parse failed: /extras/android/support
Package parse failed: /extras/google/google_play_services
Package parse failed: /extras/google/m2repository
Package parse failed: /extras/intel/Hardware_Accelerated_Execution_Manager
Package parse failed: /platform-tools
Package parse failed: /platforms/android-15
Package parse failed: /platforms/android-16
Package parse failed: /platforms/android-17
Package parse failed: /platforms/android-18
Package parse failed: /platforms/android-19
Package parse failed: /platforms/android-20
Package parse failed: /platforms/android-21
Package parse failed: /platforms/android-22
Package parse failed: /platforms/android-23
Package parse failed: /platforms/android-L
Package parse failed: /platforms/android-MNC
Package parse failed: /samples/android-22
Package parse failed: /samples/android-23
Package parse failed: /samples/android-MNC
Package parse failed: /sources/android-15
Package parse failed: /sources/android-16
Package parse failed: /sources/android-17
Package parse failed: /sources/android-18
Package parse failed: /sources/android-19
Package parse failed: /sources/android-21
Package parse failed: /sources/android-22
Package parse failed: /sources/android-23
Package parse failed: /system-images/android-15/default/armeabi-v7a
Package parse failed: /system-images/android-15/default/x86
Package parse failed: /system-images/android-16/default/armeabi-v7a
Package parse failed: /system-images/android-17/default/armeabi-v7a
Package parse failed: /system-images/android-17/default/x86
Package parse failed: /system-images/android-18/default/armeabi-v7a
Package parse failed: /system-images/android-18/default/x86
Package parse failed: /system-images/android-19/default/armeabi-v7a
Package parse failed: /system-images/android-19/default/x86
Package parse failed: /system-images/android-20/android-wear/armeabi-v7a
Package parse failed: /system-images/android-20/android-wear/x86
Package parse failed: /system-images/android-21/android-tv/armeabi-v7a
Package parse failed: /system-images/android-21/android-tv/x86
Package parse failed: /system-images/android-21/default/armeabi-v7a
Package parse failed: /system-images/android-21/default/x86
Package parse failed: /system-images/android-21/default/x86_64
Package parse failed: /system-images/android-21/google_apis/armeabi-v7a
Package parse failed: /system-images/android-21/google_apis/x86
Package parse failed: /system-images/android-21/google_apis/x86_64
Package parse failed: /system-images/android-22/android-tv/armeabi-v7a
Package parse failed: /system-images/android-22/android-tv/x86
Package parse failed: /system-images/android-22/default/armeabi-v7a
Package parse failed: /system-images/android-22/default/x86
Package parse failed: /system-images/android-22/default/x86_64
Package parse failed: /system-images/android-22/google_apis/armeabi-v7a
Package parse failed: /system-images/android-22/google_apis/x86
Package parse failed: /system-images/android-22/google_apis/x86_64
Package parse failed: /system-images/android-23/android-tv/armeabi-v7a
Package parse failed: /system-images/android-23/android-tv/x86
Package parse failed: /system-images/android-23/default/armeabi-v7a
Package parse failed: /system-images/android-23/default/x86
Package parse failed: /system-images/android-23/default/x86_64
Package parse failed: /system-images/android-23/google_apis/armeabi-v7a
Package parse failed: /system-images/android-23/google_apis/x86
Package parse failed: /system-images/android-23/google_apis/x86_64
Package parse failed: /system-images/android-L/default/armeabi-v7a
Package parse failed: /system-images/android-L/default/x86
Package parse failed: /system-images/android-L/default/x86_64
Package parse failed: /system-images/android-MNC/default/arm64-v8a
Package parse failed: /system-images/android-MNC/default/armeabi-v7a
Package parse failed: /system-images/android-MNC/default/x86
Package parse failed: /system-images/android-MNC/default/x86_64
Package parse failed: /tools
    0 packages installed.
Querying update sites for available packages...
Traceback (most recent call last):
  File "/Users/tristan/Projects/personal/android-sdk-updater/bin/android-sdk-updater", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/Users/tristan/Projects/personal/android-sdk-updater/src/bin/android-sdk-updater", line 39, in <module>
    sys.exit(main())
  File "/Users/tristan/Projects/personal/android-sdk-updater/src/bin/android-sdk-updater", line 34, in main
    update_sdk.main(android_home)
  File "/Users/tristan/Projects/personal/android-sdk-updater/src/sdk_updater/update.py", line 86, in main
    available = list_packages(android)
  File "/Users/tristan/Projects/personal/android-sdk-updater/src/sdk_updater/list.py", line 31, in list_packages
    out = subprocess.getoutput(android + ' list sdk --all --extended')
AttributeError: 'module' object has no attribute 'getoutput'
```

Looks like you're using a lot of python3 only APIs. I'm digging through and updating them as I find them. I'll see if I can get this working a little better.

### CODE REVIEW

**Comment by tadfisher** [comment]
> Looks like you're using a lot of python3 only APIs.

Yeah, I only tested on python 3.5. Is this bad behavior for a python package?

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+::
+
+    android-sdk-updater <sdk-root> [<package> ...]
+
+License
+-------
+
+::
+
+    Copyright 2015 Tad Fisher
```
Comment (tadfisher) [inline README.rst:36]: I think you're missing someone here.

File: sdk_updater/list.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
```
Comment (tadfisher) [inline sdk_updater/list.py:1]: :(

**Comment by twaddington** [comment]
> Looks like you're using a lot of python3 only APIs.

Most folks still aren't upgraded to Python3. Best practice is to be backwards compatible. There are lots of different ways to accomplish this.

File: README.rst (hunk 1 above)
Comment (twaddington) [inline README.rst:36]: This copyright text was copied verbatim from your Markdown file.

Comment (tadfisher) [inline README.rst:36]: I know :)

**Comment by tadfisher** [comment]
OK, good to know. I'll merge and give a shot at making it work on 2.7.

File: README.rst (hunk 1 above)
Comment (twaddington) [inline README.rst:36]: Oh, right. Sorry, not feeling well.

Comment (twaddington) [inline README.rst:36]: Feel free to add me if you like.

**Pull request merged**